packages/app/src/systems/Core/components/Header.tsx

```diff
@@ -1,126 +1,56 @@
-import { cssObj } from '@fuel-ui/css';
-import { Box, FuelLogo, IconButton, ButtonLink, Link } from '@fuel-ui/react';
+import { Nav } from '@fuel-ui/react';
 import { useLocation } from 'react-router-dom';
-import { IS_PREVIEW } from '~/config';
-import { useTheme } from '~/systems/Settings';
 import { Pages } from '~/types';
 
 import { removeTrailingSlash } from '../utils';
 
 export function Header() {
   const location = useLocation();
-  const { handlers } = useTheme();
-
-  const getClassName = (url: string) => {
-    return removeTrailingSlash(location.pathname) === removeTrailingSlash(url)
-      ? 'header--navItemActive'
-      : undefined;
+  const isLinkActive = (url: string) => {
+    return removeTrailingSlash(location.pathname) === removeTrailingSlash(url);
   };
 
   return (
-    <Box.Flex as="header" css={styles.root} align="center" justify="center">
-      <Box.Flex css={styles.wrapper} align="center" justify="space-between">
-        <Box.Flex gap="$4" css={styles.menu}>
-          <Link
-            href={IS_PREVIEW ? '/' : 'http://fuel.network'}
-            className="logo"
+    <Nav>
+      <Nav.Desktop>
+        <Nav.Logo />
+        <Nav.Spacer />
+        <Nav.Menu>
+          <Nav.MenuItem
+            href={Pages.bridge}
+            isActive={isLinkActive(Pages.bridge)}
+          >
+            Bridge
+          </Nav.MenuItem>
+          <Nav.MenuItem
+            href={Pages.ecosystem}
+            isActive={isLinkActive(Pages.ecosystem)}
+          >
+            Ecosystem
+          </Nav.MenuItem>
+        </Nav.Menu>
```

Comment:
We need to keep the logic that uses the `IS_PREVIEW`; They ensure that when we are on the ecosystem page, the menu items are not shown.